test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerTests.cs

```diff
@@ -147,6 +147,32 @@ public void RoundTripToken()
             TestUtilities.AssertFailIfErrors(context);
         }
 
+        // Test checks to make sure that default times are correctly added to the token
+        // upon token creation.
+        [Fact]
+        public void SetDefaultTimesOnTokenCreation()
```

Comment:
Do we have tests for SAML, SAML2, JwtSecurityToken?

Reply:
I don't think so. 
I added in a similar test for the other three handlers.